pretix_attestation_plugin/views.py

```diff
@@ -24,24 +21,19 @@ def get_form_kwargs(self):
         return kwargs
 
     def form_valid(self, form):
-        self.write_to_file(form.cleaned_data["key_file_data"])
+        self.write_to_file(self.request.FILES.get('keyfile', None))
         self.save_base_url(form.cleaned_data["base_url"])
         return super().form_valid(form)
 
-    def write_to_file(self, cleaned_data):
-        if(cleaned_data is None):
+    def write_to_file(self, uploaded_file):
+        if(uploaded_file is None):
```

Comment:
I do think it is correct to depend on `cleaned_data` which comes from the `Form` that takes care of both things: (a) display the config form, (b) validate the uploaded data!